app/views/catalog/_report_issue_form.html.erb

```diff
@@ -0,0 +1,59 @@
+<%= form_tag url_for(controller: controller_name, action: "report_issue"),

```

Comment:
@awead - the linter is crapping out on this file with the error:

```
Exception occured when processing: app/views/catalog/_report_issue_form.html.erb
If this file cannot be processed by erb-lint, you can exclude it in your configuration file.
begin_pos location 2052 is out of range for document of size 2016
```

I'm not really sure why `begin_pos` is 2052 instead of 0. Have you seen something like this before?

FWIW, I replaced the contents of this file with the contents of`_email_form.html.erb` from the Blacklight gem, and it failed with the same error, but different numbers, due to the different file size.

Reply:
Weird. Yeah, I've seen this happen sometimes. I just add the file to the exclude block in `.erblint.yml`:

```
exclude:
- '**/app/views/catalog/_report_issue_form.html.erb'
```